pkg/rpc/server/server.go

```diff
@@ -774,6 +775,14 @@ func (s *Server) contractScriptHashFromParam(param *request.Param) (util.Uint160
 	return result, nil
 }
 
+func (s *Server) getStateHeight(_ request.Params) (interface{}, *response.Error) {
+	height := s.chain.BlockHeight()
+	return &result.StateHeight{
+		BlockHeight: height,
+		StateHeight: height,
```

Comment:
It's supposed to answer with verified state height, so it's `-1` and only if stateroot is enabled.

Reply:
Returned 0 as default.